holidays/countries/turkmenistan.py

```
@@ -0,0 +1,69 @@
+    start_year = 1991 
+
+    def __init__(self, *args, **kwargs):
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self) 
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        """Add public holidays for Turkmenistan."""
+        if self._year >= 1991:
+            self._add_holiday_jan_1("New Year's Day")
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)